ood-portal-generator/spec/fixtures/ood-portal.conf.all

```diff
@@ -195,11 +195,19 @@ Listen 8080
     LuaHookLog analytics.lua analytics_handler
   </Location>
 
-  # CORS setup for API requests
-  <Location "/pun/sys/application/api">
+  # Setup for API requests
+  <Location "/my_pun_apps/sys/dashboard/api">
+    RewriteEngine On
+    # Client API auth check
+    RewriteCond "%{HTTP:X-OOD-API-ID}" !^[a-z0-9-]+$ [NC,OR]
+    RewriteCond "%{HTTP:X-OOD-API-CLIENT-SECRET}" !^[a-z0-9-]+$ [NC,OR]
+    RewriteCond "/opt/ood/api/clients/%{HTTP:X-OOD-API-ID}.%{HTTP:X-OOD-API-CLIENT-SECRET}" !-f
```

Comment:
I think this should be using the `@api_auth_root` here instead of hardcoding the path?

Reply:
Ugh, ignore my comment, I read this as the template not the ruby code.